### PR TITLE
[TBR]feat(model): implement "-c" option(config file)

### PR DESCRIPTION
**Pull request by StellarisW**
#### What type of PR is this?
feat

#### What this PR does / why we need it (en: English/zh: Chinese):

en:  implement "-c" option(config file)
zh:  增加了 model 中的 "-c" 选项，使用配置文件生成dao模型文件

### CODE REVIEW

File: config/model.go [hunk 1]
```diff
@@ -36,24 +39,102 @@ type ModelArgument struct {
...
+	tables := ctx.StringSlice(Tables)
+	onlyModel := ctx.Bool(OnlyModel)
+	outPath := ctx.String(OutDir)
+	outFile := ctx.String(OutFile)
+	withUnitTest := ctx.Bool(UnitTest)
+	modelPkgName := ctx.String(ModelPkgName)
+	fieldNullable := ctx.Bool(Nullable)
+	fieldWithIndexTag := ctx.Bool(IndexTag)
+	fieldWithTypeTag := ctx.Bool(TypeTag)
+	if configPath != "" {
```
Comment (welkeyever) [inline config/model.go:67]: How is the priority of this configuration file determined? 
Maybe add some description  both in your pr and in the code comment.

Comment (StellarisW) [reply config/model.go:68]: sure, 
priority:
command line > config file

File: config/model.go [hunk 2]
```diff
@@ -36,24 +39,103 @@ type ModelArgument struct {
...
+	outPath := ctx.String(OutDir)
+	outFile := ctx.String(OutFile)
+	withUnitTest := ctx.Bool(UnitTest)
+	modelPkgName := ctx.String(ModelPkgName)
+	fieldNullable := ctx.Bool(Nullable)
+	fieldWithIndexTag := ctx.Bool(IndexTag)
+	fieldWithTypeTag := ctx.Bool(TypeTag)
+	// priority: command line > config file
+	if configPath != "" {
+		if configFileParams, err := loadConfigFile(c.C); err == nil && configFileParams != nil {
```
Comment (welkeyever) [inline config/model.go:69]: In this scenario, loading errors should also be perceived by users - reduce the phenomenon of input and output inconsistencies.

Comment (StellarisW) [reply config/model.go:69]: yes, it should return error, i have fixed

**Comment by Duslia** [comment]
@li-jin-gou  cc. It's about gen tool.

**Pull request closed without merging**